pygef/gef.py

```diff
@@ -267,7 +269,9 @@ def __init__(self, path=None, string=None):
             )
 
         self.__dict__.update(parsed.__dict__)
-        self.df = self.df.dropna().reset_index(drop=True)
+
+        # Convert all NaN to None and drop them
+        self.df = self.df.fill_nan(None).drop_nulls()
```

Comment:
This operation `fill_nan` will be redundant if we write the functions correctly. Make sure that any computations we do not generate nans.